modules/fx/arrow-fx/src/main/kotlin/arrow/fx/IORunLoop.kt

```diff
@@ -415,6 +415,14 @@ internal object IORunLoop {
     }
 
     override operator fun invoke(either: Either<Throwable, Any?>) {
+      apply(either)
+    }
+
+    override fun resumeWith(result: Result<Any?>) {
+      invoke(result.fold(::Right, ::Left))
```

Comment:
`apply`?

Reply:
Then you're duplicating the bytecode to have a micromicromicrofemto perf improvement. Not worth it.

Reply:
duh, good point 😅

Reply:
Wait, if both functions are the same you can have one single function call, make one call the other :D This was more about the inline hahaha

Reply:
No prob, pushed into a different branch -> https://github.com/arrow-kt/arrow/pull/1993/commits/fefb020fd67e117d9fa6e1fff9ffc213b44d2ebd